src/Symfony/Component/Notifier/Bridge/Twilio/TwilioTransport.php

```diff
@@ -70,7 +71,15 @@ protected function doSend(MessageInterface $message): void
         if (201 !== $response->getStatusCode()) {
             $error = $response->toArray(false);
 
-            throw new TransportException(sprintf('Unable to send the SMS: '.$error['message'].' (see %s).', $error['more_info']), $response);
+            throw new TransportException(sprintf('Unable to send the SMS: '.$error['message'].' (see "%s").', $error['more_info']), $response);
         }
+
+        $success = $response->toArray(false);
+
+        $message = new SentMessage($message);
+        $message->setMessageId($success['sid']);
```

Comment:
What about get rid of setters and move to constructor arguments?

Reply:
Good idea!

Reply:
Changed! Thanks @Koc